system/lib/libcxxabi/src/private_typeinfo.cpp

```diff
@@ -1290,7 +1290,7 @@ __base_class_type_info::search_below_dst(__dynamic_cast_info* info,
 
 // XXX EMSCRIPTEN
 
-#ifdef __USING_EMSCRIPTEN_EXCEPTIONS__
+#ifndef __USING_WASM_EXCEPTIONS__
```

Comment:
when are the functions below used when exceptions are off? maybe add a comment.

Reply:
When some object files were compiled with exception catching enabled.. but we do not enable exception catching at a link time.   Basically exactly the thing that this PR is allowing.   I agree a comment would be good.